core/wiki/macros/tree.tid

```diff
@@ -1,53 +1,62 @@
 title: $:/core/macros/tree
 tags: $:/tags/Macro
 
-\define leaf-node(prefix)
+\define link(full-title, chunk)
```

Comment:
Hi @morosanuae , maybe it's better to use a more unique macro-name because `<<link>>` would be available everywhere because of the system tag and may be used for other things by users?

Reply:
You're right, I'll do that. Thanks @BurningTreeC.